src/main/resources/org/apache/fluo/resources/java-checkstyle.xml

```diff
@@ -170,5 +170,11 @@
       <message key="name.invalidPattern" value="Method name ''{0}'' must match pattern ''{1}''."/>
     </module>
     <module name="SingleLineJavadoc"/>
+    <module name="RegexpSinglelineJava">
```

Comment:
I copied from the Fluo pom, but did not do any additional testing. I did, however, manually inspect to ensure I understood the expression and that it made sense, as a sanity check for what already existed. The only thing I changed was to make `\.` become `[.]`, because I think that construction is more clear (escapes are hard to reason about).